adoc/SA_vars.adoc

```diff
@@ -24,9 +24,9 @@ endif::[]
 
 
 // could be "Azure","AWS","GCE" or "libvirt"
-//:cloud: Azure
+:cloud: Azure
```

Comment:
Please do not check in such changes - as this will end in a endless loop of changes